src/pipx/pipx_metadata_file.py

```diff
@@ -46,7 +46,7 @@ class PackageInfo(NamedTuple):
 
 class PipxMetadata:
     # Only change this if file format changes
-    __METADATA_VERSION__: str = "0.3"
+    __METADATA_VERSION__: str = "0.4"
```

Comment:
Should we add some documentation on what changes the different versions introduce in the future?

Reply:
Any more suggestions for the docs I added? I personally think for more details one should dig through the commit history. 

Could maybe add links to the respective PRs, but even that is very easy to find via git.